packages/@react-spectrum/s2/src/Link.tsx

```diff
@@ -83,15 +85,27 @@ const link = style<LinkRenderProps & LinkStyleProps>({
 
 function Link(props: LinkProps, ref: FocusableRef<HTMLAnchorElement>) {
   [props, ref] = useSpectrumContextProps(props, ref, LinkContext);
-  let {variant = 'primary', staticColor, isQuiet, isStandalone, UNSAFE_style, UNSAFE_className = '', styles} = props;
+  let {variant = 'primary', staticColor, isQuiet, isStandalone, UNSAFE_style, UNSAFE_className = '', styles, children} = props;
 
   let domRef = useFocusableRef(ref);
+  let isSkeleton = useContext(SkeletonContext) || false;
+  [children, UNSAFE_style] = useSkeletonText(children, UNSAFE_style);
+
+  useLayoutEffect(() => {
+    if (domRef.current) {
+      // TODO: should RAC Link pass through inert?
```

Comment:
I think so, seems like an ok thing to pass through in general and might make it easier for people to make their website/form builders instead of having to wrap all their components